src/main/java/edu/cornell/kfs/concur/businessobjects/ConcurAccountInfo.java

```diff
@@ -10,9 +10,9 @@
     protected String subObjectCode;
     protected String projectCode;
     protected String orgRefId;
+    protected boolean forPersonalCorporateCardExpense;
```

Comment:
There is no other information on the ConcurAccountInfo that can be used to determine if it is used for personal corp card. Adding a boolean value to determine that seemed like a reasonable approach.